services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None, actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__in=genres_ids, actors__in=actors_ids
+        )
+    if genres_ids:
+        return Movie.objects.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        return Movie.objects.filter(actors__in=actors_ids)
+
+    return Movie.objects.all()
```

Comment:
You can simplify your code
```suggestion
    queryset =  Movie.objects.all()
    if genres_ids:
        queryset = queryset.objects.filter(genres__in=genres_ids)

    if actors_ids:
        queryset = queryset.objects.filter(actors__in=actors_ids)

    return queryset
```